url.bs

```diff
@@ -1068,6 +1068,33 @@ code point is U+003A (:).
 <p class="note">As per the <a href=#url-writing>URL writing</a> section, only a
 <a>normalized Windows drive letter</a> is conforming.
 
+<p>A string <dfn>starts with Windows drive letter</dfn> if all of the following are true:
```

Comment:
a Windows drive letter*

Reply:
Also, to allow alternative spelling later on use `lt="start with a Windows drive letter|starts with a Windows drive letter"` here.